Index.html

```diff
@@ -33,6 +34,161 @@ <h2 class="subtitulo">I&apos;am a Software Developer</h2>
+              <ul>
+                <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+                <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+                <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+              </ul>
+            </div>
+             <a target="_blank" class="b2" href="https://www.microverse.org/">See Project</a>
+            </div>
+    
+            <div class="projects">
+              <h2 class="tituloPAPD">Profesional Art Printing Data</h2>
+              <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+              <div class="list_programs2">
+                <ul>
+                  <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+                  <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+                  <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+                </ul>
+              </div>
+              <a target="_blank" class="b2" href="https://www.microverse.org/">See Project</a>
+              </div>
+
+              <div class="projects">
+                <h2 class="tituloPAPD">Profesional Art Printing Data</h2>
+                <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <div class="list_programs2">
+                  <ul>
+                    <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+                    <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+                    <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+                  </ul>
+                </div>
+                <a target="_blank" class="b2" href="https://www.microverse.org/">See Project</a>
+                </div>
+        
+                <div class="projects">
+                  <h2 class="tituloPAPD">Profesional Art Printing Data</h2>
+                  <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                  <div class="list_programs2">
+                    <ul>
+                      <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+                      <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+                      <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+                    </ul>
+                  </div>
+                  <a target="_blank" class="b2" href="https://www.microverse.org/">See Project</a>
+                  </div>
+              
+                </div>
+
```

Comment:
| Your design |
|-----------|
|![Screenshot from 2022-10-29 18-00-26](https://user-images.githubusercontent.com/65129228/198840632-dc3d7d19-45ef-4e43-9e8b-4d382ea9a617.png) |
- [ ] Good job on implementing the previous CR feedback, but some issues were missed when you implemented it. All the projects in the section should be aligned with a CSS grid property and the project also be aligned with a CSS property. Kindly fix :wrench: . For further reference, [check this](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns)

Example
```
.project1 {
  display: grid;
  grid-template-columns: auto;
}

.projects {
  display: flex;
}
```